controllers/metrics.go

```diff
@@ -23,6 +23,26 @@ import (
 
 // Prometheus metrics gauges
 var (
+	DBAvailableMetric = prometheus.NewGaugeVec(
```

Comment:
Don't you need to [register these like the others](https://github.com/opendatahub-io/data-science-pipelines-operator/blob/main/controllers/metrics.go#L69)?

Reply:
done